From cef17ae41f0bf38b31b7c8ce400c1e116424aed8 Mon Sep 17 00:00:00 2001
From: prasannas <prasannas@ami.com>
Date: Mon, 16 Oct 2023 15:01:00 +0530
Subject: [PATCH 124/124] Added Success Message For Clearing Dump Logs

Testcase:
1)Clearing Dump Logs

Post call
https://10.0.136.98/redfish/v1/Managers/bmc/LogServices/Dump/Actions/LogService.ClearLog

Body:
{
}

Response:
{
    "@Message.ExtendedInfo": [
        {
            "@odata.type": "#Message.v1_1_1.Message",
            "Message": "The request completed successfully.",
            "MessageArgs": [],
            "MessageId": "Base.1.16.0.Success",
            "MessageSeverity": "OK",
            "Resolution": "None"
        }
    ]
}
---
 redfish-core/lib/log_services.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/redfish-core/lib/log_services.hpp b/redfish-core/lib/log_services.hpp
index a517851..0ec639e 100644
--- a/redfish-core/lib/log_services.hpp
+++ b/redfish-core/lib/log_services.hpp
@@ -1034,6 +1034,7 @@ inline void clearDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             messages::internalError(asyncResp->res);
             return;
         }
+        messages::success(asyncResp->res);
     },
         "xyz.openbmc_project.Dump.Manager",
         "/xyz/openbmc_project/dump/" + dumpTypeLowerCopy,
-- 
2.9.5

